3-box-model/style.css

```diff
@@ -0,0 +1,34 @@
+.footer {
+    position: fixed;
+    border: 2px orange solid;
+    background-color: black;
+    color: white;
+    width: 100%;
+    bottom: 0;
+}
+
+.header {
+    position: sticky;
+    border: 2px green solid;
+    background-color: black;
+    color: white;
+}
+
+.nav {
+    position: absolute;
+    border: 2px blue solid;
+    padding-bottom: 50px;
+    padding-right: 100px;
```

Comment:
so, has this element 200px of width? can you double check using the developer tools?